pyro/distributions/von_mises_3d.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import absolute_import, division, print_function
+
+import math
+
+import torch
+from torch.distributions import constraints
+
+from pyro.distributions import TorchDistribution
+
+
+class VonMises3D(TorchDistribution):
+    """
+    Spherical von Mises distribution.
+    https://en.wikipedia.org/wiki/Von_Mises%E2%80%93Fisher_distribution
```

Comment:
nit: this would be better as a hyperlink. also we should have this for all distributions or none imo

Reply:
Sure, I'll add explanation and remove the link.